ert3/console/__init__.py

```diff
@@ -97,11 +97,11 @@ def _export(workspace_root, experiment_name):
     if not _experiment_have_run(experiment_root):
         raise ValueError("Cannot export experiment that has not been carried out")
 
-    with open(experiment_root / ".storage.json") as f:
-        data = json.load(f)
-
+    input_data = ert3.storage.get_input_data(workspace_root, experiment_name)
+    output_data = ert3.storage.get_output_data(workspace_root, experiment_name)
+    data = _reformat_input_output({"input": input_data, "output": output_data})
```

Comment:
It is a bit awkward to pack the data in a dict to unpack it at the other side of the function afterwards 🤷 I suggest we change the function to take `input` and `output` as arguments instead of `data`. But it can wait until you have something working ;)

Reply:
I think this is a good idea :) I guess we just used the functions as they were to start with to see if we can get the tests running